doc/classes/WorkerThreadPool.xml

```diff
@@ -1,10 +1,50 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="WorkerThreadPool" inherits="Object" version="4.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
+		A singleton that allocates some [Thread]s on startup, used to offload tasks to these threads.
 	</brief_description>
 	<description>
+		The [WorkerThreadPool] singleton allocates a set of [Thread]s (called worker threads) on project startup and provides methods for offloading tasks to them. This can be used for simple multithreading without having to create [Thread]s.
+		Tasks hold the [Callable] to be run by the threads. [WorkerThreadPool] can be used to create regular tasks, which will be taken by one worker thread, or group tasks, which can be distributed between multiple worker threads. Group tasks execute the [Callable] multiple times, which makes them useful for iterating over a lot of elements, such as the enemies in an arena.
+		Here's a sample on how to offload an expensive function to worker threads:
+		[codeblocks]
+		[gdscript]
+		var enemies = [] # An array of enemies that must be processed every frame.
+
+		func process_enemy_ai(enemy_index):
+		    var processed_enemy = enemies[enemy_index]
+		    # Expensive logic...
+
+		func _process(delta):
+		    var task_id = WorkerThreadPool.add_group_task(process_enemy_ai, enemies.size())
+		    # Other code...
+		    WorkerThreadPool.wait_for_group_task_completion(task_id)
+		    # Other code that depends on the enemy AI already being processed.
+		[/gdscript]
+		[csharp]
+		private Node[] enemies = [] // An array of enemies that must be processed every frame.
```

Comment:
```suggestion
		private Node[] _enemies = {}; // An array of enemies that must be processed every frame.
```

For empty arrays it's better to use `Array.Empty<Node>()` instead of `{}`. However, I assume this array is not meant to be empty. Using `Godot.Collections.Array` of `System.List` may be better, but I don't know where the enemies are suppose to come from.